specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -411,58 +389,25 @@ would need to be pre-set to a well-known string that defines Push Triggers.
 
 namespace Microsoft.Windows.PushNotifications
 {
+    [feature(Feature_PushNotifications)]
     // Event args for the Push payload.
     runtimeclass PushNotificationReceivedEventArgs
     {
-        // Initialize using the IBackgroundInstance: used specifically for the Background Activation scenario
-        static PushNotificationReceivedEventArgs CreateFromBackgroundTaskInstance(Windows.ApplicationModel.Background.IBackgroundTaskInstance backgroundTask);
-
-        // Initialize using the PushNotificationEventArgs from Windows: used specifically for in-memory event handling when app is already in foreground
-        static PushNotificationReceivedEventArgs CreateFromPushNotificationReceivedEventArgs(Windows.Networking.PushNotifications.PushNotificationReceivedEventArgs args);
-
         // The Push payload
         byte[] Payload { get; };
 
-        // Gets a deferral to run under specific modes like low power mode
+        // Gets a deferral to allow push notification processing even if the system goes into low power mode.
```

Comment:
This is **only** needed for allowing notification processing in low power-mode right? 

If so, let's re-word as "// Gets a deferral to allow notification processing when the system is in low-power mode"

Reply:
Deferrals are not always related to low power. They're a way for the app to say "I will need to continue working even after the event handler that sent me this returns, so please don't consider me done until I complete the deferral."  I'd remove the "into low power mode" text and say:

```c++
// Gets a deferral for when the app needs to continue processing after the event handler returns.
// Complete the deferral when the app has finished handling the notification.
```